app/pulp/app/tests/testapp/apps.py

```diff
@@ -0,0 +1,10 @@
+"""
+This is where we connect our plugin to Pulp. Having `PulpPluginAppConfig` as a parent class
+identifies this app as a Pulp plugin. We specify the location our app so it may be
+installed.
+"""
+from pulp.app.apps import PulpPluginAppConfig
+
+
+class TestAppConfig(PulpPluginAppConfig):
+    name = 'pulp.app.tests.testapp'
```

Comment:
This would normally includ the app's "label", which in this case is just "testapp". For a normal plugin, where the app would be pulp_rpm.app, for example, this would just be "pulp_rpm". tl;dr add a `label = 'testapp'`. I'm not sure if I called this out in the docs, but it's explained in PulpAppConfig's comments.